### DISCUSSION TITLE
Not able to run and debug the code in Xcode 13.1

### DISCUSSION THREAD

**Discussion by tniraj7** [touchlab/xcode-kotlin#41] [Q&A]
Hi,  the `./setup.sh`script runs and after that when I load the Xcode and add 
the `commonMain` and `iOSMain` folders into the project, I cannot see the kotlin file icons 
as shown in the `README.md` file. Also the Xcode warns about loading the bundle.

I am not able to debug the code. Is there any place where all the steps are documented ?
Thank you.

**Comment by kpgalligan** [comment]
When Xcode updates you need to grab the new UUID and we can add it to the list. See here: https://github.com/touchlab/xcode-kotlin#xcode-updates

We (touchlab) are using 13.0 internally, so we haven't bumped the 13.1 version. We generally take PRs from the community as other people tend to be more aggressive on Xcode updates. See merged PRs:

<img width="436" alt="Screen Shot 2021-11-15 at 7 05 47 AM" src="https://user-images.githubusercontent.com/68384/141779289-d172ab84-0a16-44f5-8a84-b0bcd7db60ae.png">

So, if you'd like, add a PR, or at least run the command to get the UUID and we'll add it.

> Also the Xcode warns about loading the bundle.

That is expected. Xcode is generally not extendable, and even the bits we can make Xcode angry.

**Reply by tniraj7** [reply]
this was the UUID that I got `8BAA96B4-5225-471B-B124-D32A349B8106`

**Reply by Praveend437** [reply]
Xcode 13.1 version UUID is 7A3A18B7-4C08-46F0-A96A-AB686D315DF0. Added same in Contents/info.plist, uuid's array.

Me also facing issue to debug Kotlin code from Xcode, Any suggestions please?
